core/src/banking_stage.rs

```diff
@@ -1988,54 +1949,6 @@ impl BankingStage {
         process_transactions_summary
     }
 
-    // Returns the number of packets that were filtered out for
-    // no longer being valid (could be too old, a duplicate of something
-    // already processed, etc.)
```

Comment:
Reading this description, I was looking for where in the function we filter out transactions that are no longer valid `could be too old, a duplicate of something already processed, etc.)`

It looks like I accidentally deleted this logic in my PR:
https://github.com/solana-labs/solana/commit/870ac80b7948f22dca0eceea0061b1f74c64f56b#diff-bb8464c2f02c8863fb60e6e29f023f3e6da93a93fbb9199a70a44c9c1b41c7f3L1897-L1902

This should probably probably be added back in? This would mean the new forwarding bucket logic would have to make these sanity checks as well if we deleted it here